app/controllers/glossary_terms_controller.rb

```diff
@@ -8,12 +8,7 @@ class GlossaryTermsController < ApplicationController
   # ---------- Actions to Display data (index, show, etc.) ---------------------
 
   def index
-    # Index should be paged with alpha and number tabs
-    # See https://www.pivotaltracker.com/story/show/167657202
-    # Glossary should be query-able
-    # See https://www.pivotaltracker.com/story/show/167809123
-    includes = @user ? { thumb_image: :image_votes } : :thumb_image
-    @glossary_terms = GlossaryTerm.includes(includes).order(:name)
+    filter_index? ? index_filtered : index_full
   end
```

Comment:
If we are going to make `pattern_search` available to the glossary, we may want to do a very short `index_dispatch` for this controller. @JoeCohen I will go with your preference, since you've been working on those dispatches. Please feel free to suggest a commit here and below.